settings/ui/tests/run-acceptance-test.sh

```diff
@@ -46,7 +46,15 @@ export SERVER_HOST=${SERVER_HOST:-https://localhost:9200}
 export BACKEND_HOST=${BACKEND_HOST:-https://localhost:9200}
 export TEST_TAGS=${TEST_TAGS:-"not @skip"}
 
-yarn run acceptance-tests "$1"
+cucumber-js --retry 1 \
```

Comment:
it does not find cucumber-js
I tried to use cucumber-js from `${TEST_INFRA_DIRECTORY}/node_modules/@cucumber/cucumber/bin/cucumber-js` (after `yarn install` there) and also from `${WEB_PATH}/tests/acceptance//node_modules/@cucumber/cucumber/bin/cucumber-js`

and in both cases I get the message
```
Error: 
      You appear to be executing an install of cucumber (most likely a global install)
      that is different from your local install (the one required in your support files).
      For cucumber to work, you need to execute the same install that is required in your support files.
      Please execute the locally installed version to run your tests.
```

So I think we do depend on the deps in the acceptance tests folder